src/Symfony/Component/BrowserKit/CookieJar.php

```diff
@@ -143,7 +135,7 @@ public function updateFromSetCookie(array $setCookies, $uri = null)
      * @param Response $response A Response object
      * @param string   $uri      The base URL
      */
-    public function updateFromResponse(Response $response, $uri = null)
+    public function updateFromResponse(Response $response, string $uri = null)
```

Comment:
Since the `$uri` parameter accepts null, the correct typehint here should be `?string`

```suggestion
    public function updateFromResponse(Response $response, ?string $uri = null)
```